src/controllers/payment.controller.ts

```diff
@@ -1,38 +1,66 @@
 import { inject } from 'inversify'
-import { controller, httpGet } from 'inversify-express-utils'
+import { controller, httpGet, httpPost } from 'inversify-express-utils'
 import { RedirectResult } from 'inversify-express-utils/dts/results'
 import { v4 as uuidv4 } from 'uuid'
 import BaseController from './base.controller'
 
+import PaymentMapper from 'app/mappers/payment/payment.mapper'
 import ApplicationStatus from 'app/models/dto/applicationStatus.enum'
+import DissolutionGetPaymentUIData from 'app/models/dto/dissolutionGetPaymentUIData'
 import DissolutionGetResponse from 'app/models/dto/dissolutionGetResponse'
+import PaymentSummary from 'app/models/dto/paymentSummary'
 import Optional from 'app/models/optional'
 import DissolutionSession from 'app/models/session/dissolutionSession.model'
-import { PAYMENT_URI, SEARCH_COMPANY_URI } from 'app/paths'
+import { HOW_DO_YOU_WANT_TO_PAY_URI, PAYMENT_REVIEW_URI, SEARCH_COMPANY_URI } from 'app/paths'
 import DissolutionService from 'app/services/dissolution/dissolution.service'
 import PaymentService from 'app/services/payment/payment.service'
 import SessionService from 'app/services/session/session.service'
+import TYPES from 'app/types'
 
-@controller(PAYMENT_URI)
+interface ViewModel {
+  paymentSummary: PaymentSummary
+}
+
+@controller(PAYMENT_REVIEW_URI)
 export class PaymentController extends BaseController {
 
   public constructor(
-    @inject(SessionService) private session: SessionService,
-    @inject(DissolutionService) private service: DissolutionService,
-    @inject(PaymentService) private paymentService: PaymentService
+    @inject(SessionService) private sessionService: SessionService,
+    @inject(DissolutionService) private dissolutionService: DissolutionService,
+    @inject(PaymentService) private paymentService: PaymentService,
+    @inject(PaymentMapper) private paymentMapper: PaymentMapper,
+    @inject(TYPES.CHS_API_KEY) private CHS_API_KEY: string,
```

Comment:
this can be injected into the service instead, or even the client? The API injects this into the client